geps/gep-1856.md

```diff
@@ -0,0 +1,46 @@
+# GEP 1856: Egress
+Issue: [#1856](https://github.com/kubernetes-sigs/gateway-api/issues/1856)
+
+Status: Provisional
```

Comment:
At this point I would really like to see this GEP merge in, for posterity. But as per several discussions the Gateway API project is not in a great position to support this use case at the time.

I have been working with Phillip Klatte (couldn't find his Github?) on the possibility that the [Kubernetes Network Interface (KNI)](https://github.com/kubernetes/enhancements/pull/4477) might help to get us there. For the moment, rather than leaving this open without movement long term, and without closing it and missing the opportunity to document the intention I would like to make a proposal:

What if we call it `Withdrawn` for now?

```suggestion
Status: Withdrawn
```

This effectively means it's on hold until further notice. LMKWYT :thinking: 
```